cmd/node/node.go

```diff
@@ -294,7 +294,12 @@ func (app *SpacemeshApp) initServices(nodeID types.NodeId, swarm service.Service
 	}
 
 	//todo: put in config
-	atxdb := activation.NewActivationDb(atxdbstore, mdb, 1000)
+	iddbstore, err := database.NewLDBDatabase(dbStorepath+"ids", 0, 0)
```

Comment:
we've already fixed this bug in our branch, but for better UT the DB should be received from whom ever calls this function, in case of UT you dont need persistent DB like LDB